src/gui/properties/propertieswidget.cpp

```diff
@@ -144,7 +145,7 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     m_ui->vBoxLayoutPeerPage->addWidget(m_peerList);
     // Tab bar
     m_tabBar = new PropTabBar(nullptr);
-    m_tabBar->setContentsMargins(0, 5, 0, 0);
+    m_tabBar->setContentsMargins(0, 3, 0, 3);
```

Comment:
I have changed int from `m_tabBar->setContentsMargins(0, 5, 0, 0);` in previous commit revision to this, it looks better I guess
new one
![image](https://user-images.githubusercontent.com/34717789/80923452-893dc800-8da1-11ea-8c4c-16d5d100fe8a.png)

Reply:
I would prefer `(0, 5, 0, 5)`, looks better to me.